### PR TITLE
Creat String module

### PR DESCRIPTION
**Pull request by Z-Najdaoui**
-added method ucwords in String model that capitalizes the first letter of every word in a string given

### CODE REVIEW

File: syntax/string.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const String = {
+  /**
+   * The ucwords() method capitalizes the first letter of every word in a string.
+   * @param {string} str
+   * @returns {string}
+   * @example
+   * range("hello world") // output:"Hello World"
```
Comment (whybe7) [inline syntax/string.js:7]: the name of the `example` function is not compatible with the actual `function` just change it and you're good to go 👌🏻

**Review by whybe7 — COMMENTED** [review]
yoo, @Z-Najdaoui  it's a wonderful ✨ job there you made :octocat:

File: syntax/string.js (hunk 1 above)
Comment (whybe7) [reply syntax/string.js:7]: in the example, you still have the `range` function but the actual name of the function is `ucwords`.

Comment (Z-Najdaoui) [reply syntax/string.js:7]: It's a copying problem, actually. I am using the same Doxygen as your array module.

Comment (whybe7) [reply syntax/string.js:7]: yes it is better for making doc usage of the code/functions

**Review by whybe7 — APPROVED** [review]
This is a wonderful piece of work ✨👌🏻

File: syntax/string.js (hunk 1 above)
Comment (whybe7) [reply syntax/string.js]: that was amaizing

**Pull request merged**